src/index.js

```diff
@@ -1,26 +1,11 @@
 import './style.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-// eslint-disable-next-line
-import { addItem, displayItem } from '../modules/taskFunction.js';
-// eslint-disable-next-line
-let arrayOfTasks = JSON.parse(localStorage.getItem('arrayOfTasks')) || [];
-const toDoList = document.querySelector('.toDoList');
-const taskInput = document.querySelector('.taskInput');
-const addBtn = document.getElementById('addBtn');
 
-const deleteTask = (index) => {
-  arrayOfTasks = arrayOfTasks.filter((task) => task.index !== index);
-  arrayOfTasks.forEach((task, index) => {
-    task.index = index + 1;
-  });
-  localStorage.setItem('arrayOfTasks', JSON.stringify(arrayOfTasks));
-  displayItem();
-};
+// fetching containers
+import { displayItem, addItem } from './modules/addItem.js';
+
+const addBtn = document.getElementById('addBtn');
 
 // event listeners
 document.addEventListener('DOMContentLoaded', displayItem);
```

Comment:
- [OPTIONAL] Kindly make sure when the page reloads we can still see that the task was checked but at this moment the completed task can not show their styles after the page was reloaded.

 - ### YOUR APP BEHAVIOR

  ![chrome-capture-2023-5-10 (1)](https://github.com/Chandan-devs-tech/To-Do-list/assets/87197412/66e4d3c1-93b2-4d04-9c4e-98af3c193e8d)